site/docs/api/table-options.md

```diff
@@ -1132,6 +1134,20 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
 
 - **Example:** [Search On Enter Key](https://examples.bootstrap-table.com/#options/search-on-enter-key.html)
 
+## searchSelector
+
+- **Attribute:** `data-search-selector`
+
+- **Type:** `Boolean|String`
+
+- **Detail:**
+
+  If this option isset (must be a valid dom selector e.g. `#customSearch`), the found dom element (should be a `input` element) will be used as table search instead of the build in search input.
```

Comment:
Typos, 
...If this options **is set**...
should be **an input**
...instead of the **buit-in**

Reply:
Will be changed :-)

Reply:
done